.npmignore

```diff
@@ -0,0 +1,9 @@
+/.vscode
+/src
```

Comment:
```suggestion

```

I want to keep the source files in built packages, which may helpful when using with sourcemap.